viam-cartographer.go

```diff
@@ -42,11 +40,13 @@ import (
 	goutils "go.viam.com/utils"
 	"go.viam.com/utils/pexec"
 	"go.viam.com/utils/protoutils"
-
-	"github.com/viamrobotics/viam-cartographer/internal/grpchelper"
 )
 
 var (
+	// Model is the model name of cartographer.
+	Model = resource.NewModel("viam", "slam", "cartographer")
+	// BinaryLocation contains the name of the cartographer grpc server.
+	BinaryLocation                = "carto_grpc_server"
```

Comment:
[optional] If we decide to go with the alternative in https://github.com/viamrobotics/viam-orb-slam3/pull/19#discussion_r1140388585, should we do the same here?